src/entities/product/product.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { Exclude } from 'class-transformer';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn
+} from 'typeorm';
+
+@Entity()
+export class Product {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ unique: true })
+  name: string;
+
+  @Column('decimal')
+  price: number;
+
+  @Column()
+  description: string;
+
+  @Column()
+  stock: number;
+
+  @CreateDateColumn()
+  created_at: Date;
```

Comment:
A general question here, I see that you use sneak_case convention to name some files and variables. Is there any particular reason to use sneak_case instead of camelCase for those? In TypeScript sneak_case is only used (commonly) to name constants like `DB_PORT`

Reply:
I was just using the example given by typeorm, ill change it to camelCase in the next commit!
Thanks